src/simulation/monsters/low/g-m/ManiacalMonkey.ts

```diff
@@ -0,0 +1,28 @@
+import LootTable from '../../../../structures/LootTable';
+import SimpleMonster from '../../../../structures/SimpleMonster';
+
+export const ManiacalMonkeyTable = new LootTable()
+	.every('Small ninja monkey bones')
+
+	/* Weapons and armour */
+	.add('Steel scimitar', 1, 2)
+	.add('Maple shortbow', 1, 1)
+
+	/* Materials */
+	.add('Oak plank', 1, 3)
+	.add('Grimy guam leaf', 1, 2)
+
+	/* Other */
+	.add('Bass', 2, 4)
+	.add('Banana', 1, 2)
+	.add('Antipoison(2)', 1, 2)
+	.add('Prayer potion(1)', 1, 2)
+	.add('Adamant arrow(p++)', 1, 1)
+	.add('Rope', 1, 1);
+
+export default new SimpleMonster({
+	id: 6803,
```

Comment:
Where did you get this ID from? It looks wrong and that it should be `7118`

Reply:
got it from a osrs monster ID website, but yea I think the ID I used was for another instance of the monkey in a quest or something. changing it now